common/Unit.cpp

```diff
@@ -60,7 +60,7 @@ UnitBase** UnitBase::linkAndCreateUnit(UnitType type, const std::string& unitLib
     DlHandle = dlopen(unitLibPath.c_str(), RTLD_GLOBAL|RTLD_NOW);
     if (!DlHandle)
     {
-        LOG_ERR("Failed to load " << unitLibPath << ": " << dlerror());
+        LOG_ERR("Failed to load unit-test lib " << dlerror());
```

Comment:
Why hiding the unitLibPath?

Reply:
Turns out `dlerror()` already includes the path, so it was a duplicate.

Reply:
Ah, makes sense. Thanks!